dists/install.sh

```diff
@@ -122,14 +121,7 @@ main() {
   fi
   echo "DONE"
   status "Extracting...   "
-  case $ext in
-    zip)
-      unzip -d "${tmpdir}" "${tmpdir}/${archive_name}"
-      ;;
-    tar.gz)
-      tar -C "${tmpdir}" -x -f "${tmpdir}/${archive_name}"
-      ;;
-  esac
+  tar -C "${tmpdir}" -x -f "${tmpdir}/${archive_name}"
```

Comment:
Are we sure `tar` is available on all MacOS?

Reply:
I asked @ksol and it ran on his mac.

Reply:
pretty sure it's been available for quite some time